indico/modules/rb/client/js/modules/bookRoom/filters/RecurrenceForm.jsx

```diff
@@ -35,7 +35,7 @@ export default class RecurrenceForm extends FilterFormComponent {
       this.stateChanger('type')(e.target.value);
     };
     this.onTypeChange = this.stateChanger('type');
-    this.onNumberChange = this.stateChanger('number', num => Math.abs(parseInt(num, 10) || 1));
+    this.onNumberChange = this.stateChanger('number', num => Math.abs(parseInt(num, 10)));
```

Comment:
what's the reason for this change?

Reply:
From mobile you can't delete nor modify the `1` that is there by default. It's true that this makes the the form more error prone as you can try to submit it without a value but I couldn't think of any other solution that is mobile-friendly, do you have any other suggestion?